### PR TITLE
[Frontend Setup] [Client] Set up preliminary routing with react-router-dom

### PR DESCRIPTION
**Pull request by testsnake**
Added react-router-dom to the project with very simple test routes and an error page

### CODE REVIEW

**Review by susansli — CHANGES_REQUESTED** [review]
This is a good start, but I think there must have been a misunderstanding of some sort. The task was to create routes for all the pages that you saw in the wireframes. For example, home, portfolio, settings, just to name the few in the bottom nav bar. There should also be others like onboarding related... etc. Please let me know if you have any more questions.

File: apps/augury/src/main.tsx
Comment (susansli) [inline apps/augury/src/main.tsx:1]: You also want the router in a separate module, it should be imported in `main.tsx`

File: apps/augury/src/app/error-page.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { useRouteError } from 'react-router-dom';
+
+export default function ErrorPage() {
+  const error = useRouteError();
```
Comment (susansli) [inline apps/augury/src/app/error-page.tsx:4]: I understand this is for demo, but let's remove it and create an actual 404 later.

File: apps/augury/src/app/error-page.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  console.error(error);
+
+  return (
+    <div id="error-page">
```
Comment (susansli) [inline apps/augury/src/app/error-page.tsx:8]: Also as a btw, we should be using the Chakra equivalents, e.g. `Box`

**Review by susansli — CHANGES_REQUESTED** [review]

File: apps/augury/src/app/error-page.tsx (hunk 1 above)
Comment (testsnake) [reply apps/augury/src/app/error-page.tsx:4]: So it should just error out without a page for now?

**Review by susansli — APPROVED** [review]
LG2M, let's just keep our route names consistent and change `/NotFound` to `/not-found` (we should stick to lowercase if possible). I'll approve the PR, but would like you to make the change before merging to dev. Thanks! :)

**Pull request merged**